packages/nvidia_nat_a2a/src/nat/plugins/a2a/server/oauth_middleware.py

```
@@ -0,0 +1,121 @@
+            logger.warning("Missing or invalid Authorization header")
+            return JSONResponse({
+                "error": "unauthorized", "message": "Missing or invalid Bearer token"
+            },
+                                status_code=401)
+
+        token = auth_header[7:]  # Strip "Bearer "
+
+        # Validate token using NAT's validator
+        try:
+            result = await self.validator.verify(token)
+        except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)